### DISCUSSION TITLE
LEFT function didn't work

### DISCUSSION THREAD

**Discussion by lionelcanon** [dbeaver/dbeaver#29283] [General]
Hello,
why left function sql doesn't work in DBeaver ?

SELECT LEFT('abcdefghij',1) FROM dual;

Thanks a lot :)

**Comment by LonwoLonwo** [comment]
Hello.
What database do you use?
What error do you have, or how exactly it doesn't work?

**Reply by lionelcanon** [reply]
hello,
Im use an oracle database.

Error its : SQL Error [904] [42000]: ORA-00904: "LEFT" : identificateur non valide

**Reply by LonwoLonwo** [reply]
The Oracle database does not support the LEFT function.
You can take a look at this answer:
https://stackoverflow.com/a/45495735

**Comment by lionelcanon** [comment]
thanks a lot, it's ok with substr
